webapp/src/actions/index.js

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This files is a bit inconsistent, have arrow functions, regular functions, `async/await` approach and `.then`s, I would prefer to have a consistent implementation.

As a reference, in mattermost, for actions, we normally use regular functions and we prefer `async/await` over `then`s (when is possible).